pylint/utils.py

```diff
@@ -1188,11 +1188,11 @@ def _ini_format(stream, options, encoding):
     """format options using the INI format"""
     for optname, optdict, value in options:
         value = _format_option_value(optdict, value)
-        help = optdict.get('help')
-        if help:
-            help = _normalize_text(help, line_len=79, indent='# ')
+        help_opt = optdict.get('help')
+        if help_opt:
+            help_opt = _normalize_text(help, line_len=79, indent='# ')
```

Comment:
The tests are currently failing because of this, it should be `help_opt`, not `help`